compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt

```diff
@@ -28,345 +27,345 @@ import org.jetbrains.skiko.*
 
         /** 'V' key. */
-        actual val V: Key = skikoKeyToKey(SkikoKey.KEY_V)
+        actual val V = Key(86)
 
         /** 'W' key. */
-        actual val W: Key = skikoKeyToKey(SkikoKey.KEY_W)
+        actual val W = Key(87)
 
         /** 'X' key. */
-        actual val X: Key = skikoKeyToKey(SkikoKey.KEY_X)
+        actual val X = Key(88)
 
         /** 'Y' key. */
-        actual val Y: Key = skikoKeyToKey(SkikoKey.KEY_Y)
+        actual val Y = Key(89)
 
         /** 'Z' key. */
-        actual val Z: Key = skikoKeyToKey(SkikoKey.KEY_Z)
+        actual val Z = Key(90)
 
         /** ',' key. */
-        actual val Comma: Key = skikoKeyToKey(SkikoKey.KEY_COMMA)
+        actual val Comma = Key(188)
 
         /** '.' key. */
-        actual val Period: Key = skikoKeyToKey(SkikoKey.KEY_PERIOD)
+        actual val Period = Key(190)
 
         /** Left Alt modifier key. */
-        actual val AltLeft: Key = skikoKeyToKey(SkikoKey.KEY_LEFT_ALT)
+        actual val AltLeft = Key(18)
 
         /** Right Alt modifier key. */
-        actual val AltRight: Key = skikoKeyToKey(SkikoKey.KEY_RIGHT_ALT)
+        actual val AltRight = Key(225)
 
         /** Left Shift modifier key. */
-        actual val ShiftLeft: Key = skikoKeyToKey(SkikoKey.KEY_LEFT_SHIFT)
+        actual val ShiftLeft = Key(16)
 
         /** Right Shift modifier key. */
-        actual val ShiftRight: Key = skikoKeyToKey(SkikoKey.KEY_RIGHT_SHIFT)
+        actual val ShiftRight = Key(-2147483632) // 0x80000000.toInt() or 16
 
         /** Tab key. */
-        actual val Tab: Key = skikoKeyToKey(SkikoKey.KEY_TAB)
+        actual val Tab = Key(9)
 
         /** Space key. */
-        actual val Spacebar: Key = skikoKeyToKey(SkikoKey.KEY_SPACE)
+        actual val Spacebar = Key(32)
 
         /** Enter key. */
-        actual val Enter: Key = skikoKeyToKey(SkikoKey.KEY_ENTER)
+        actual val Enter = Key(13)
 
         /**
          * Backspace key.
          *
          * Deletes characters before the insertion point, unlike [Delete].
          */
-        actual val Backspace: Key = skikoKeyToKey(SkikoKey.KEY_BACKSPACE) // Key(KeyEvent.VK_BACK_SPACE)
+        actual val Backspace = Key(8) // Key(KeyEvent.VK_BACK_SPACE)
 
         /**
          * Delete key.
          *
          * Deletes characters ahead of the insertion point, unlike [Backspace].
          */
-        actual val Delete: Key = skikoKeyToKey(SkikoKey.KEY_DELETE)
+        actual val Delete = Key(46)
 
         /** Escape key. */
-        actual val Escape: Key = skikoKeyToKey(SkikoKey.KEY_ESCAPE)
+        actual val Escape = Key(27)
 
         /** Left Control modifier key. */
-        actual val CtrlLeft: Key = skikoKeyToKey(SkikoKey.KEY_LEFT_CONTROL)
+        actual val CtrlLeft = Key(17)
 
         /** Right Control modifier key. */
-        actual val CtrlRight: Key = skikoKeyToKey(SkikoKey.KEY_RIGHT_CONTROL)
+        actual val CtrlRight = Key(-2147483631) // 0x80000000.toInt() or 17
```

Comment:
```suggestion
        actual val CtrlRight = Key(0x80000000.toInt() or 17)
```

It's even better to Move `0x80000000` to named "mask" const

Reply:
@MatkovIvan what will benefit from the fact that this values are masked? Does it helps in certain scenarios? 
I'm not arguing - I just want to see a bigger picture, to understand what was initial intention behind this

Reply:
To get clear readable code like:
```kt
actual val CtrlLeft = Key(LEFT_MASK or 17)
actual val CtrlRight = Key(RIGHT_MASK or 17)
```